lab2/src/include/Statistics.h

```diff
@@ -3,7 +3,10 @@
 
 #include <cstdlib>
 #include <vector>
-
+/**
+ * Statistics
+ * handles statistics calculations for vector<double> types.
+ */
```

Comment:
No, I meant that maybe we need to add documentation to what mean / variance / etc.. does.